core/domain/suggestion_registry.py

```diff
@@ -162,6 +163,21 @@ def validate(self):
                     'Expected final_reviewer_id to be a string, received %s' %
                     type(self.final_reviewer_id))
 
+        if (
+                not user_id_migration.verify_user_id_correct(self.author_id) and
```

Comment:
@vojtechjelinek is this ok? (I thought we were going to delete user_id_migration after the migration.)